extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1414,6 +1441,11 @@ export class AmpA4A extends AMP.BaseElement {
     if (this.sentinel) {
       mergedAttributes['data-amp-3p-sentinel'] = this.sentinel;
     }
+    const browserSupportsSandbox = this.win.HTMLIFrameElement &&
```

Comment:
Let's move this to when you set this.shouldSandbox_

Reply:
Done.